src/Form/DataList.svelte

```diff
@@ -0,0 +1,138 @@
+
+<script>
+  import Icon from "../Icons/Icon.svelte";
+  import { createEventDispatcher } from "svelte";
+
+  export let value = "";
+  export let name = undefined;
+  export let thin = false;
+  export let secondary = false;
+  export let outline = false;
+  export let disabled = false;
+  export let editable = false;
```

Comment:
afaik, DataList should always be editable... thats what sets it apart from Select. We should probably remove this prop

Reply:
Yeah, makes sense was just copy pasta from `Select`